### PR TITLE
docs(change log): Add hitTest improvements & update bug/enh list

### PR DESCRIPTION
**Pull request by lboyd93**
This should probably wait to be merged until the next update (Monday?).

@annelfitz could you review please? Thanks!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,12 +4,25 @@ The `next` version of 4.29 is now available. Planned release date is February 20
 
 ![Current build version](https://img.shields.io/npm/v/arcgis-js-api/next?label=Current%20build)
 
+## Performance updates
+The [heatmapRendererCreator.createRenderer()](https://developers.arcgis.com/javascript/latest/api-reference/esri-smartMapping-renderers-heatmap.html#createRenderer) method for generating a [HeatmapRenderer](https://developers.arcgis.com/javascript/latest/api-reference/esri-renderers-HeatmapRenderer.html) is 100x faster than it was in version 4.28.
```
Comment (steveoh) [inline CHANGELOG.md:8]:
```suggestion
## Performance updates

The [heatmapRendererCreator.createRenderer()](https://developers.arcgis.com/javascript/latest/api-reference/esri-smartMapping-renderers-heatmap.html#createRenderer) method for generating a [HeatmapRenderer](https://developers.arcgis.com/javascript/latest/api-reference/esri-renderers-HeatmapRenderer.html) is 100x faster than it was in version 4.28.
```

**Review by annelfitz — APPROVED** [review]
🚀 🚀 🚀

File: CHANGELOG.md (hunk 1 above)
Comment (lboyd93) [reply CHANGELOG.md]: Thanks @steveoh 😄

**Comment by lboyd93** [comment]
FYI @ubatsukh

**Pull request merged**